ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/contract/ContractCacheOptions.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.ethereum.core.contract;
+
+import picocli.CommandLine;
+
+public class ContractCacheOptions {
```

Comment:
Would move under unstable

Reply:
See previous note about circular dependencies and package-by-feature, unless you are proposing an unstable packge being added here?